src/caches/rosenbrock_caches.jl

```diff
@@ -67,7 +67,7 @@ function alg_cache(alg::Rosenbrock23,u,rate_prototype,uEltypeNoUnits,uBottomElty
   J,W = iip_generate_W(alg,u,uprev,p,t,dt,f,uEltypeNoUnits)
   tmp = zero(rate_prototype)
   atmp = similar(u, uEltypeNoUnits)
-  tab = Rosenbrock23ConstantCache(real(uBottomEltypeNoUnits),identity,identity,nothing,nothing,nothing)
+  tab = Ros23ConstantCache(real(uBottomEltypeNoUnits))
```

Comment:
I think I would prefer the name `Rosenbrock23Tableau`.

Reply:
I agree, but then we need to correct all the tab struct in `\tableaus` as well as places calling the tabs in `\caches`

Reply:
Sure, there might be some other structs which probably should be renamed as well but that can be done in a different PR, I guess. For now, IMO it's sufficient to name the new struct in a reasonable way :smiley: